pager/lib.h

```diff
@@ -81,4 +81,6 @@ void mutt_buffer_strip_formatting(struct Buffer *dest, const char *src, bool str
 
 void mutt_clear_pager_position(void);
 
+bool config_init_pager(struct ConfigSet *cs);
+
 #endif /* MUTT_PAGER_H */
```

Comment:
Please can you tidy up a few things I forgot to do in my libpager commit:

- Fix the `#include` guard name to match the file: `MUTT_PAGER_LIB_H`

- Update `po/POTFILES.in` to match the new files/locations
  Here's a [one-liner command](https://neomutt.org/run/release/translate#popotfilesin) to do it

- Add a Doxygen library header block
  Here's the Sidebar's:

https://github.com/neomutt/neomutt/blob/8ff8cd7f5e5f773c257522218583e9d36324c237/sidebar/lib.h#L25-L40

Reply:
done